### DISCUSSION TITLE
画像の埋め込み方法について、もっと親切な説明が欲しい

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#312] [要望・アイデア・不具合]
ここ数日間、投稿に添付する画像の大きさの調整に苦慮しておりました。画像の幅のピクセル数をある程度以下に下げれば、それに比例して画像の表示の幅は小さくできますが、あまり解像度が良くありません。解像度を高くしたまま、表示される画像の大きさを小さくしたかったのです。

　しかし、「Markdown記法チートシート」の「画像埋め込み」や、「ヘルプ」の「画像のアップロード方法」を見てもその方法は書いてありません。小さな画像を含んだ他の方の記事を探し回り、そのマークダウンのソースを参考に、やっと目的の方法があることを知りました。

　その結果、図の埋め込みは `![代替テキスト](画像のURL)` 形式だけでなく、`<img width="幅" src="画像のURL">` 形式でも可能であることが分かりました。

　しかし、Qiita から公式に説明されていないところを見ると、現在は「非推奨」で将来は使えなくなる可能性がある方法なのかな？とも思えます。もしそうでなければ、是非とも説明に加えていただけると助かります。htmlダグは使える物と使えない物が混在しているので、使えるものは明示していただかないと混乱します。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご意見ありがとうございます🙏
将来使えなくなる可能性はゼロではありませんが、
画像のhtmlタグに関しては、Markdown記法チートシートに追記しておきます！

**Reply by tsubolabo** [reply]
本日、Markdown記法チートシートに追記されているのを確認できました。
どうもありがとうございました。
